backend/prisma/schema.prisma

```diff
@@ -6,30 +6,31 @@ generator client {
 }
 
 datasource db {
-  provider = "postgresql"
+  provider = "mongodb"
   url      = env("DATABASE_URL")
 }
 
-model TreeNode {
-  address   String
-  balance    Int
-  path      String[]
-  indices   String[]
-  root      String
-  createdAt DateTime  @default(now())
+model Leaf {
+  address     String   @map("_id")
+  usdValue    Int
+  merkleProof String[]
+  createdAt   DateTime @default(now())
 
-  @@unique([ address, root])
+  @@id([address])
 }
 
-model Tree {
-  root        String
-  createdAt   DateTime  @default(now())
+model Group {
+  merkleRoot     String   @map("_id")
+  name           String
+  threshold      Int
+  erc20Addresses String[]
```

Comment:
I am unsure if we need to save erc20 addresses since they will be available from the leaf object

Reply:
@christopherdancy -- where are the ERC20addresses on the leaf object? The leaf is only the address, balance, and proof currently.